index.html

```diff
@@ -23,67 +24,79 @@
           <div class="navbar__logo">
             <a href="#" class="logo link fz-18 fw-700 1h-20">Daphine</a>
           </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <ul>
+          <img
+            id="humberger"
+            class="humberger-toggle"
+            src="./humberger.svg"
+            alt="menu icon"
+          />
+          <img src="./assets/cancel.svg" class="cancel hide" alt="cancel" />
+          <div class="menu-icon menu-toggle">
+            <ul class="menu-icon-ul">
```

Comment:
- [ ] Let's kindly make sure our navigation has an anchor tag and link us to the correct section when clicked for example when the user click/tap portfolio it takes him to the projects section. also, it should close the mobile menu!

Reply:
* Issue resolved